airflow-core/src/airflow/serialization/serialized_objects.py

```diff
@@ -3784,16 +3784,33 @@ class LazyDeserializedDAG(pydantic.BaseModel):
     last_loaded: datetime.datetime | None = None
 
     NULLABLE_PROPERTIES: ClassVar[set[str]] = {
-        "is_paused_upon_creation",
+        # Non attr fields that should be nullable, or attrs with a different default
         "owner",
+        "owner_links",
         "dag_display_name",
+        "has_on_success_callback",
+        "has_on_failure_callback",
+        
```

Comment:
```suggestion
```

This should fix static check failure: https://github.com/apache/airflow/actions/runs/17831596554/job/50698808007?pr=55833